src/components/common/FilterDropdown.jsx

```diff
@@ -0,0 +1,31 @@
+import Dropdown from 'react-bootstrap/Dropdown';
+import CustomMenu from './FilterDropdownMenu';
+
+function FilterDropdown({ currElement, data, callbackFunc }) {
+  return (
+    <Dropdown onSelect={(eventKey) => callbackFunc(eventKey)}>
+      <Dropdown.Toggle id="dropdown-custom-components">{currElement}</Dropdown.Toggle>
+
+      <Dropdown.Menu as={CustomMenu}>
+        {data.map((value, index) => {
+          if (value === currElement) {
+            return (
+              // eslint-disable-next-line react/no-array-index-key
+              <Dropdown.Item key={index} eventKey={value} active>
```

Comment:
when we get the proper school data objects, you can replace the key with the schools id to address the eslint error